src/components/MagicCodeInput.js

```diff
@@ -111,6 +111,11 @@ function MagicCodeInput(props) {
             setFocusedIndex(0);
             inputRefs.current[0].focus();
         },
+        resetFocus() {
+            setInput('');
+            setFocusedIndex(0);
```

Comment:
Instead of below 2 lines we can directly call focus() to avoid duplicate code

Reply:
@WikusKriek  This seems to be fine IMO if we are unable to call within imperative functions, compared to having duplicate code. We can name it as `focusMagicCodeInput` similar to blur